File: example/ApiTest/Python/sessions/3_list_sessions.py
import requests

url = "http://127.0.01/api/v1/chats/36734bf8aee011ef9eb50242ac120003/sessions?name=change session name&id=b745827eaee411efa65f0242ac120003"

payload = {}
headers = {
  'Authorization': 'Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: example/ApiTest/C#/sessions/3_list_sessions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

namespace RagflowAPI
{
    internal class Program
    {
        static async Task Main(string[] args)
        {
            var client = new HttpClient();
            var request = new HttpRequestMessage(HttpMethod.Get, "http://127.0.01/api/v1/chats/36734bf8aee011ef9eb50242ac120003/sessions?name=change session name&id=b745827eaee411efa65f0242ac120003");
            request.Headers.Add("Authorization", "Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm");
            var response = await client.SendAsync(request);
            response.EnsureSuccessStatusCode();
            Console.WriteLine(await response.Content.ReadAsStringAsync());

        }
    }
}
